app/controllers/idv/capture_doc_controller.rb

```diff
@@ -15,7 +15,7 @@ class CaptureDocController < ApplicationController
       step_url: :idv_capture_doc_step_url,
       final_url: :root_url,
       flow: Idv::Flows::CaptureDocFlow,
-      analytics_id: Analytics::DOC_AUTH,
+      analytics_id: ::Analytics::DOC_AUTH,
```

Comment:
Reusing the "Analytics" name introduces some awkwardness with referencing the class in `Idv` module, since one must choose between referencing `::Analytics` and `Idv::Analytics`.

Reply:
What if we called it `Idv::AnalyticsExtender` or something?

Reply:
Yeah that could help 👍

Reply:
or `Idv::SupplementedAnalytics`

Reply:
I worked around this indirectly by revising the implementation approach of the class to be a decorator in 91437d2ac, so now it's `Idv::AnalyticsDecorator`.